### PR TITLE
[DROOLS-4674] Temporary remove background Tab

### PR DESCRIPTION
**Pull request by yesamer**
@dupliaka @gitgabrio @danielezonca

Temporary removing  background grid which is still not working, in order to don't include it for 7.29.0 release.

https://issues.jboss.org/browse/DROOLS-4674

### CODE REVIEW

**Comment by dupliaka** [comment]
jenkins execute full downstream build

**Comment by dupliaka** [comment]
Checked manually.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java [hunk 1]
```diff
@@ -391,13 +389,7 @@ public void onOverviewSelected() {
      */
     @Override
     public void addBackgroundPage(final ScenarioGridWidget scenarioGridWidget) {
-        addPage(new PageImpl(scenarioGridWidget, ScenarioSimulationEditorConstants.INSTANCE.backgroundTabTitle()) {
```
Comment (gitgabrio) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java]: @yesamer 
Please leave this code as it is and - instead - remove method invocation

Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java]: @gitgabrio Done

**Comment by yesamer** [comment]
jenkins retest this please

**Comment by yesamer** [comment]
@danielezonca can you please merge it?

**Comment by yesamer** [comment]
@kiegroup/gatekeepers can you please merge it on master?

**Pull request merged**